### ISSUE TITLE
Show test result summary in the console

### ISSUE THREAD

**Issue by renato04** [scanapi/scanapi#497]
### Description the feature
Show the test result summary in the console and return a status process error.
The objective is if I put the Scan API on a pipeline this pipeline has to be stopped if there is an error. 

### Is your feature request related to a problem?
No.

### Do you have any suggestions on how to add this feature in scanapi ?
No.

**Comment by Pradhvan** [comment]
@rafaquelhodev we do have a [GitHub Action](https://github.com/scanapi/github-action) for scanapi. I am not sure how something similar can be done for any other CI.  

@camilamaia any thoughts ?

For the console part I guess it would be really nice if we can use [rich](https://github.com/willmcgugan/rich) to have similar report shown in the terminal. Though I am not sure how much comprehensive report on the console would be compared to jinja template one. But can definitely try it out.

Happy to work on this feature. 😄

**Comment by renato04** [comment]
@Pradhvan I think for comprehension it can show just the summary like this:
Run: 3 tests Passed 2 Failed 1
Some information about the error
Return -1 as system code
Stop the execution

What do you think about it?

**Comment by Pradhvan** [comment]
@renato04 yes, something like this. Should we make two console outputs? 

* one where the entire report is shown with a pretty looking console output using rich. This can be a separate CLI flag too like the browser one. 
* Second would be the one that you mentioned and would be specifically for CI pipelines, does not have to be pretty just need to show the info that you share above 

```
Total: 3 tests Passed 2 Failed 1
Some information about the error
Return -1 as system code
```

**Comment by renato04** [comment]
@Pradhvan it can has two output types. 
I think each of them should have their own issue.

**Comment by Pradhvan** [comment]
@renato04 yes, that's what I was suggesting. 😄

Waiting for @scanapi/core-team @scanapi/maintainers to reply on what these folks think.

**Comment by camilamaia** [comment]
Hey!

I like the idea of the feature, thanks @renato04! Definitely we need it when the `-nr, --no-report` flag is passed. But I think without the flag would also be cool. Maybe something like: "this is the summary, if you want to check the full report, open the file ..."

> The objective is if I put the Scan API on a pipeline this pipeline has to be stopped if there is an error.

This already happens. If ScanAPI is added to a pipeline and any test fails, it will break the pipeline. This happens because we set the exit code to 1 when a test failure happens.

https://github.com/scanapi/scanapi/blob/5f10b7e36458f935256a38edf5290706f68610d1/scanapi/session.py#L32

Also, the user can upload the report as an asset in the CI. This way, it is available the full report. Example:

https://github.com/scanapi/scanapi/actions/runs/265531139

It is an example using github actions, but it is also possible with different CIs, like circleci https://circleci.com/docs/2.0/artifacts/

https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts

What do you think?

**Comment by renato04** [comment]
Hey @camilamaia !

> This already happens. If ScanAPI is added to a pipeline and any test fails, it will break the pipeline. This happens because we set the exit code to 1 when a test failure happens.

I didn't know that. It's awesome.

> Maybe something like: "this is the summary, if you want to check the full report, open the file ..."

I think this will be great.

@Pradhvan  What do you think about it?

**Comment by astenstrasser** [comment]
closed by #542
